genesis/initialize_test.go

```diff
@@ -0,0 +1,34 @@
+package genesis_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/OffchainLabs/prysm/v6/config/params"
+	"github.com/OffchainLabs/prysm/v6/genesis"
+	"github.com/OffchainLabs/prysm/v6/genesis/embedded"
+	"github.com/OffchainLabs/prysm/v6/testing/require"
+)
+
+func TestInitialize(t *testing.T) {
+	require.NoError(t, genesis.Initialize(t.Context(), "testdata"))
+	require.Equal(t, params.MainnetName, params.BeaconConfig().ConfigName)
+
+}
```

Comment:
This seems to only test the happy path where an embedded genesis is found

<img width="1009" height="555" alt="Image" src="https://github.com/user-attachments/assets/8d88ecf4-31e3-4c33-b196-bba16e0fda52" />

Reply:
Resolved in feb7d4f5643047e48370cadd7929ba22a2b3d0ae